examples/terraform-asg-scp-example/main.tf

```diff
@@ -26,7 +26,7 @@ provider "aws" {
 resource "aws_launch_template" "sample_launch_template" {
   name_prefix            = var.instance_name
   image_id               = data.aws_ami.ubuntu.id
-  instance_type          = "t2.micro"
+  instance_type          = var.instance_type
```

Comment:
This is not directly related to this PR, but we were getting intermittent test failures when we picked an instance type that happened not to be available in all AZs in an AWS region. To make tests more stable, I made these configurable, and used `GetRecommendedInstanceType` in our test code to pick a type that is available in all AZs.